proxy/http/HttpTransactHeaders.cc

```diff
@@ -750,12 +752,10 @@ HttpTransactHeaders::insert_via_header_in_request(HttpTransact::State *s, HTTPHd
   via_string += nstrcpy(via_string, s->http_config_param->proxy_hostname);
 
   *via_string++ = '[';
-  /* I thought we should use the transaction local outgoing IP address but
-     that makes cycle detection (which is the point) unrealiable. We must
-     use the same value every time to be sure.
-  */
-  memcpy(via_string, Machine::instance()->ip_hex_string, Machine::instance()->ip_hex_string_len);
-  via_string += Machine::instance()->ip_hex_string_len;
+  if (uuid != NULL) {
```

Comment:
I don't think the machine uuid can ever be NULL, but harmless to check I guess :).